src/Elliptic/DiscontinuousGalerkin/DgOperator.hpp

```diff
@@ -626,6 +626,16 @@ struct DgOperatorImpl<System, Linearized, tmpl::list<PrimalFields...>,
         // This is the sign flip that makes the operator _minus_ the Laplacian
         // for a Poisson system
         *operator_applied_to_vars *= -1.;
+      } else if (formulation == ::dg::Formulation::StrongLogical) {
+        // Strong divergence but with the Jacobian moved into the divergence:
+        //   div(F) = 1/J_p \sum_q (D_\hat{i})_pq J_q (J^\hat{i}_i)_q (F^i)_q.
+        const auto logical_fluxes = transform::first_index_to_different_frame(
+            primal_fluxes, det_times_inv_jacobian);
+        logical_divergence(operator_applied_to_vars, logical_fluxes, mesh);
+        *operator_applied_to_vars *= -1.;
```

Comment:
I'd combine the negation and the multiply when possible so we only run through the memory once. Even better would be having a bool template parameter on `logical_divergence` to `-=` instead of `+=` but that's a bigger change so optional.

Reply:
Combined the negation with the Jacobian multiply. The `-=` can be done too but I didn't decide immediately how best to do it.